src/ontology/mondo-edit.obo

```diff
@@ -147175,6 +147182,7 @@ synonym: "isolated Klippel-Feil syndrome caused by mutation in GDF6" EXACT [MOND
 synonym: "Kfs" RELATED [OMIM:118100]
 synonym: "KFS1" RELATED ABBREVIATION [MONDO:Lexical, OMIM:118100]
 synonym: "Klippel-Feil Syndrome" BROAD [NORD:1336]
```

Comment:
This should have been removed, something is going a bit wrong

Reply:
> There is no existing qc check that an exact synonym can not be the same as a broad synonym.

the checked called `duplicate_synonym_scope` (a standard ODK check) should take care of that, at least if memory serves right! Need to test again with the QC checks you added, thanks you.

> Also, I see the nord.robot.tsv that it contains a synonym type column, was that expected to be added?

Yes, that was expected.